### PR TITLE
#933: Optik TAN backend-frontend integration

### PR DESCRIPTION
**Pull request by ivaniuminov**
https://github.com/adorsys/open-banking-gateway/issues/933

### CODE REVIEW

File: opba-protocols/sandboxes/xs2a-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/testsandbox/BasicTest.java [hunk 1]
```diff
@@ -84,4 +84,21 @@ void startTheSandboxInDocker() {
         // Loop forever.
         await().forever().until(() -> false);
     }
+
+    /**
+     * Not really a test, but just launches entire sandbox for you.
```
Comment (valb3r) [inline opba-protocols/sandboxes/xs2a-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/testsandbox/BasicTest.java:89]: Please fix JavaDoc comment, it is not starting sandbox, it starts WireMock that imitates Sandbox

Comment (ivaniuminov) [reply opba-protocols/sandboxes/xs2a-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/testsandbox/BasicTest.java]: JavaDoc updated

File: opba-protocols/sandboxes/xs2a-sandbox/src/main/resources/docker-compose-with-wiremock.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: "3.4"
+
+services:
+  xs2a-sandbox-postgres:
+    image: rodolpheche/wiremock:2.27.1
+    ports:
+      - 8090:8080
```
Comment (valb3r) [inline opba-protocols/sandboxes/xs2a-sandbox/src/main/resources/docker-compose-with-wiremock.yml:7]: This should use Sandbox port

Comment (ivaniuminov) [reply opba-protocols/sandboxes/xs2a-sandbox/src/main/resources/docker-compose-with-wiremock.yml:7]: Port was changed to 20014

**Pull request merged**